src/sentry/templates/sentry/vercel-ui-hook.vercel

```diff
@@ -1,8 +1,18 @@
 <Page>
 {% block main %}
-    <Box display="flex" justifyContent="space-between">
+    <Box display="flex" flexDirection="column">
+      <H2>Setup Instructions for Sentry</H2>
       <P>
-        • <Link href="{{ link }}">Add to project for {{ org }}</Link><BR />
+        In order to get the most out of Sentry for your organization {{ org }}, you should make sure to do the following if you haven't so yet:
```

Comment:
Looks good! Nit on this sentence though, maybe it could be "...make sure to do the following if you haven't done so yet" or just "...make sure to do the following if you haven't yet"

Reply:
@ceorourke Fixed!